src/Indice.Features.Cases.AspNetCore/Controllers/AdminAccessRulesController.cs

```diff
@@ -38,6 +37,16 @@ public async Task<IActionResult> GetAccessRules([FromQuery] ListOptions<GetAcces
         return Ok(AccessRuless);
     }
 
+    /// <summary>Get Access rules for the specified case.</summary>
+    /// <param name="caseId"></param>
+    [HttpGet("/case/{caseId:Guid}")]
```

Comment:
This should be an extension to the case itself. So the correct path should follow /cases/{caseId}/access-rules

Reply:
Updated